### PR TITLE
[SMALLFIX] Read the UNDERFS_HDFS configuration from tachyon conf

### PR DESCRIPTION
**Pull request by yupeng9**
Fixed a bug in HdfsUnderFileSystem

### CODE REVIEW

**Comment by haoyuan** [comment]
could this be unit tested?

File: conf/tachyon-env.sh.template [hunk 1]
```diff
@@ -101,6 +101,8 @@ export TACHYON_JAVA_OPTS+="
   -Dtachyon.worker.tieredstore.level0.dirs.path=${TACHYON_RAM_FOLDER}
   -Dtachyon.worker.tieredstore.level0.dirs.quota=${TACHYON_WORKER_MEMORY_SIZE}
   -Dtachyon.underfs.address=${TACHYON_UNDERFS_ADDRESS}
+  -Dtachyon.underfs.glusterfs.volumes=${TACHYON_UNDERFS_GLUSTERFS_VOLUMES}
```
Comment (apc999) [inline conf/tachyon-env.sh.template]: so do we require `$TACHYON_UNDERFS_GLUSTERFS_VOLUMES` and `$TACHYON_UNDERFS_GLUSTERFS_MOUNTS` always set?
If they are not set, will this be a problem?

File: conf/tachyon-env.sh.template [hunk 2]
```diff
@@ -101,6 +101,8 @@ export TACHYON_JAVA_OPTS+="
...
+  -Dtachyon.underfs.glusterfs.mounts=${TACHYON_UNDERFS_GLUSTERFS_MOUNTS}
```
Comment (calvinjia) [inline conf/tachyon-env.sh.template]: Is this change related?

File: conf/tachyon-env.sh.template (hunk 1 above)
Comment (yupeng9) [inline conf/tachyon-env.sh.template]: It's useful when GlusterFS is used as under FS. Without defining those variables, the options will be configured as null and therefore ignored.

We can also create a template file for gluster only, but then we may need to synchronize the changes among the templates.

**Comment by yupeng9** [comment]
Not a bug. `UNDERFS_HDFS_CONFIGURATION` is set by `prepareConfiguration`.
